activity_calendar/templates/activity_calendar/activity_page_base.html

```diff
@@ -0,0 +1,203 @@
+                    <!-- Event Description -->
+                    <div class="col align-self-center">
+                        <p id="event-description">{{ activity.description }}</p>
+                    </div>
+                </div>
+
+                <div class="d-flex">
+                    <!-- Text Icon -->
+                    <div class="col-xs-2">
+                        <img
+                                src="{% static 'octicons/clippy.svg' %}"
+                                alt=""
+                                height="16"
+                                width="16"
+                        />
+                    </div>
+                    <!-- Event Description -->
+                    <div class="col align-self-center">
+                        <p id="event-status">
+                            {% if subscriptions_open %}
+                                {% if num_total_participants >= max_participants and max_participants != -1 %}
+                                    Activity is full.
+                                {% else %}
+                                    Activity is open for sign-ups
+                                {% endif %}
+                            {% else %}
+                                Subscriptions will open on {% get_opening_time "j E, H:i" %}
+                            {% endif %}
+                        </p>
+                    </div>
+                </div>
+                {% if activity.subscriptions_required %}
+                    <strong>NB:</strong> You need to register for this activity before you can join!
+                {% endif %}
+
+                <div style="height: 4em;" class="d-none d-md-flex">
+
+                </div>
+            {% endblock activity-info %}
+        </div>
+        <div class="col-12 col-md-5">
+            {% if request.user.is_authenticated %}
+                {% block sign-up-status %}
+
+                {% endblock sign-up-status %}
+            {% else %}
+                <div class="card border border-info activity-wide">
+                    <div class="row">
+                        <div class="col-12 col-sm-7 col-md-12">
+                            <div class="d-block p-2">
+                                You are currently not logged in. Log in if you want to subscribe to this activity.
+                            </div>
+                        </div>
+                        <div class="col-12 col-sm-5 col-md-12">
+                            <a href="{% get_login_url %}"
+                               class="btn btn-success btn-block">
+                                Log in
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            {% endif %}
+            <div class="alert alert-warning">
+                Corona protocol is active! Got symptoms? Stay home!<br>
+                <a href="https://beta.kotkt.nl/nextcloud/s/JpGkgdMLCGX4nHL/download">
```

Comment:
Outside this PR's scope: we should probably stop putting links to this protocol in the code

Reply:
O yeah, it was a temporary placeholder for early in the project. Gonna make a modal for that. Or at least the Corona rules (not the sign-up instructions, that should, if designed well, flow well)